<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewUser</class>
 <widget class="QWidget" name="NewUser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>555</width>
    <height>365</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 0, 220);</string>
  </property>
  <widget class="QLabel" name="l_newuser">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>10</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 24pt &quot;.SF NS Text&quot;;
background: transparent;
color: rgb(234, 239, 238);
</string>
   </property>
   <property name="text">
    <string>New User</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_firstname">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>230</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(207, 211, 211);
border-style:outset;
border-radius:10px;
font: 14pt &quot;Arial&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Enter your First Name</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_lastname">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>80</y>
     <width>229</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(207, 211, 211);
border-style:outset;
border-radius:10px;
font: 14pt &quot;Arial&quot;;</string>
   </property>
   <property name="placeholderText">
    <string>Enter your Last Name</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_phone">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>230</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(207, 211, 211);
border-style:outset;
border-radius:10px;
font: 14pt &quot;Arial&quot;;</string>
   </property>
   <property name="placeholderText">
    <string>Enter your Phone Number </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_email">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>140</y>
     <width>229</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(207, 211, 211);
border-style:outset;
border-radius:10px;
font: 14pt &quot;Arial&quot;;</string>
   </property>
   <property name="placeholderText">
    <string>Enter your Email Address </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_username">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>200</y>
     <width>230</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(207, 211, 211);
border-style:outset;
border-radius:10px;
font: 14pt &quot;Arial&quot;;</string>
   </property>
   <property name="placeholderText">
    <string>Enter Username</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>200</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(207, 211, 211);
border-style:outset;
border-radius:10px;
font: 14pt &quot;Arial&quot;;</string>
   </property>
   <property name="placeholderText">
    <string>Enter Password</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_submit">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>270</y>
     <width>159</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(160,0,0);
border-style:outset;
border-radcolor: rgb(0, 0, 0);
background-color:rgb(175,238,238);
border-style:outset;
border-radius:10px;
font: 14pt &quot;Arial&quot;;ius:10px;
font: 14pt &quot;Arial&quot;;</string>
   </property>
   <property name="text">
    <string>Submit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Back">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>320</y>
     <width>159</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(160,0,0);
border-style:outset;
border-radcolor: rgb(0, 0, 0);
background-color:rgb(175,238,238);
border-style:outset;
border-radius:10px;
font: 14pt &quot;Arial&quot;;ius:10px;
font: 14pt &quot;Arial&quot;;</string>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>0</y>
     <width>571</width>
     <height>371</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>C:/Users/bestm/Desktop/IMG/Pokemon.jpg</pixmap>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>l_newuser</zorder>
  <zorder>txt_firstname</zorder>
  <zorder>txt_lastname</zorder>
  <zorder>txt_phone</zorder>
  <zorder>txt_email</zorder>
  <zorder>txt_username</zorder>
  <zorder>lineEdit</zorder>
  <zorder>btn_submit</zorder>
  <zorder>Back</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
